docassemble/MAEvictionMoratorium/data/questions/prose_mtd_non_emergency_merge.yml

```diff
@@ -577,28 +542,51 @@ content: |
 ---
 id: send user email
 code: |
-  users[0].message_succeeded = send_email(to=users[0], template=initial_email_template)
+  if remote_signers.number() > 0 and defined( 'users[0].email' ) and users[0].email:
+    users[0].message_succeeded = send_email(to=users[0], template=initial_email_template)
   users[0].was_notified_of_cosigner_messages = True
 ---
-id: user email before signatures
+id: after remote signatures requested
 template: initial_email_template
 subject: |
-  Your Motion to Dismiss was sent to ${ codefendants[i] }
+  Your Motion to Dismiss was sent to ${ comma_and_list( remote_signers ) }
 content: |
-
   ${ users[0] },
 
-  % if codefendants.number() == 1:
-  We e-mailed ${ codefendants[i] } to ask for their signature on the Motion to Dismiss.
+  % if remote_signers.number() == 1:
+  We e-mailed ${ remote_signers } to ask for their signature on the Motion to Dismiss.
 
-  We will e-mail you when ${ codefendants[i] } has signed.  But, you can [check if ${ codefendants[i] } signed the motion](${ interview_url_action('check_status', party_id=users[0].id) }).
+  We will e-mail you when ${ remote_signers } has signed, but you can also [check if ${ remote_signers } signed the motion](${ interview_url_action('check_status', party_id=users[0].id) }).
   
   % else:
-  We e-mailed ${ codefendants[i] } to ask them to sign your Motion to Dismiss.
+  We e-mailed ${ comma_and_list( remote_signers ) } to ask them to sign your Motion to Dismiss.
 
-  We will e-mail you when everyone has signed.  But, you can [check if ${ codefendants[i] } have signed the motion](${ interview_url_action('check_status', party_id=users[0].id) }).
+  We will e-mail you when everyone has signed, but you can also [check if ${ comma_and_list( remote_signers ) } have signed the motion](${ interview_url_action('check_status', party_id=users[0].id) }).
   % endif
 ---
+# TODO: Check if user email is going to be required in this flow
+code: |
+  background_action('send_final_email')
+  final_email_sent = True
+---
+event: send_final_email
+code: |
+  if remote_signers.number() > 0 and defined( 'users[0].email' ) and users[0].email:
```

Comment:
User should always get final notification email if possible (#50), so remove the check for `remote_signers`. Instead change language of email to allow for no codefendants and also talk about all cosigners in the email body if there are any.

"Your Motion to Dismiss a Non-Essential Eviction was signed by x, y, and z/is ready."